include/ideep/operators/conv.hpp

```diff
@@ -129,7 +129,11 @@ struct conv_deconv_utils {
       // Set scales to op_attr
       int src_scale_mask = 0;
       op_attr.set_scales(DNNL_ARG_SRC, src_scale_mask, {1.0f / src_scales_in[0]});
-      auto wei_scale_mask = utils::conv_weight_scale_mask(weights_scales_in.size(), groups > 1, is_deconv);
+      // tensor::get_dim(0) returns internal_dims[0] * internal_dims[1] for grouped weight
+      // i.e., g * oc_per_group for conv and g * ic_per_group for deconv
+      int oc_per_group = is_deconv ? weight_grouped.get_dim(1) : weight_grouped.get_dim(0) / groups;
```

Comment:
should you divide `groups` on `weight_grouped.get_dim(1)` too?

Reply:
No. For grouped deconv, weight format is `gio...`. `get_dim(0)` returns `g * i` and `get_dim(1)` returns `o`, which is desired.

Reply:
Can you clarify this a bit in the comment? The current comment doesn't read straightforward to me.

Reply:
OK. I have added more details in the comment.